### ISSUE TITLE
java.lang.NullPointerException pressing details in the call log

### ISSUE THREAD

**Issue by 532910** [BelledonneCommunications/linphone-android#830]
```
11-19 03:29:38.940 25774 25774 D AndroidRuntime: Shutting down VM
--------- beginning of crash
11-19 03:29:38.941 25774 25774 E AndroidRuntime: FATAL EXCEPTION: main
11-19 03:29:38.941 25774 25774 E AndroidRuntime: Process: org.linphone, PID: 25774
11-19 03:29:38.941 25774 25774 E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.equals(java.lang.Object)' on a null object reference
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at org.linphone.contacts.l.a(LinphoneContact.java:41)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at org.linphone.history.b.a(HistoryDetailFragment.java:26)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at org.linphone.history.b.onCreateView(HistoryDetailFragment.java:30)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at android.app.Fragment.performCreateView(Fragment.java:2522)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1298)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at android.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1576)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1637)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at android.app.BackStackRecord.executeOps(BackStackRecord.java:807)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at android.app.FragmentManagerImpl.executeOps(FragmentManager.java:2405)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at android.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2200)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at android.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2155)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at android.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2062)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at android.app.FragmentManagerImpl.executePendingTransactions(FragmentManager.java:818)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at org.linphone.activities.b.a(MainActivity.java:38)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at org.linphone.history.HistoryActivity.b(HistoryActivity.java:12)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at org.linphone.history.a$a.onClick(HistoryAdapter.java:1)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at android.view.View.performClick(View.java:6597)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at android.view.View.performClickInternal(View.java:6574)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at android.view.View.access$3100(View.java:778)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at android.view.View$PerformClick.run(View.java:25906)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at android.os.Handler.handleCallback(Handler.java:873)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:99)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:193)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:6718)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
11-19 03:29:38.941 25774 25774 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
11-19 03:29:38.943  2359  4086 W ActivityManager:   Force finishing activity org.linphone/.history.HistoryActivity
11-19 03:29:38.949  2359  2475 I ActivityManager: Showing crash dialog for package org.linphone u0
```

4.2.0-beta.32+98add9b2

![image](https://user-images.githubusercontent.com/14617699/69105827-9d08a280-0a7d-11ea-983c-cc2fd606404c.png)

**Comment by 532910** [comment]
Full logs are sent via e-mail.

**Comment by Viish** [comment]
Should be fixed in 383c54f817d45009c08822410e821d17bdc1a913

**Comment by 532910** [comment]
Looks so, anyway I can't crash it more pressing that three dots.
